src/services/print.js

```diff
@@ -193,18 +193,30 @@ ngeo.Print.prototype.encodeImageLayer_ = function(arr, layer) {
 
 /**
  * @param {Array.<MapFishPrintLayer>} arr Array.
- * @param {ol.layer.Image} layer Layer.
+ * @param {ol.layer.Tile|ol.layer.Image} layer Layer.
  * @private
  */
 ngeo.Print.prototype.encodeImageWmsLayer_ = function(arr, layer) {
-  goog.asserts.assertInstanceof(layer, ol.layer.Image);
+  var url;
   var source = layer.getSource();
-  goog.asserts.assertInstanceof(source, ol.source.ImageWMS);
-  var url = source.getUrl();
+
+  if (source instanceof ol.source.ImageWMS) {
+    goog.asserts.assertInstanceof(layer, ol.layer.Image);
+    goog.asserts.assertInstanceof(source, ol.source.ImageWMS);
+    url = source.getUrl();
+  } else {
+    goog.asserts.assertInstanceof(layer, ol.layer.Tile);
+    goog.asserts.assertInstanceof(source, ol.source.TileWMS);
+    url = source.getUrls()[0];
+  }
+
   var params = source.getParams();
   var object = /** @type {MapFishPrintWmsLayer} */ ({
     baseURL: url,
     imageFormat: 'FORMAT' in params ? params['FORMAT'] : 'image/png',
+    customParams: {
+      transparent: true
+    },
```

Comment:
It is compulsory to print multiple layers.

Say I want to print layer1 (let's assume it is the background) and layer2 (rivers for instance). Without the transparent option, I will only see the image of layer2 (so the rivers) and not the background since the image of layer2 is the rivers with blank pixel between the rivers.

Should I add a comment to make this clearer?